web/viewer.html

```diff
@@ -171,6 +171,19 @@
           </div>
         </div>  <!-- findbar -->
 
+        <div class="editorParamsToolbar hidden doorHangerRight" id="editorHighlightParamsToolbar">
```

Comment:
It seems that the tab-indexes for the various editor-toolbars have now become entangled.
Please re-order the tab-indexes for *all* editor-toolbars to agree with their order in the UI.